.circleci/config.yml

```diff
@@ -25,7 +25,7 @@
         - run: sudo apt-get install cppcheck
         - run:
             name: cppcheck
-            command: cppcheck *.cpp *.h
+            command: cppcheck all *.cpp *.h
```

Comment:
`--enable=all` kapcsolóval lehetne bekapcsolni minden ellenőrzést. Most egy `all` nevű fájlt is megpróbál ellenőrizni, de azért nem szól, ha nem találja.

A performance hibákat még jó lenne javítani, a style hibákat nem kell. Úgyhogy `all` helyett elég a `performance` szint bekapcsolása.

A másik, hogy a cppcheck alapból 0-val tér vissza megtalált hibák esetén is, így a CiCI nem fog tudni a hibákról. Meg kell adni az `--error-exitcode=` kapcsolóval, hogy milyen hibakóddal térjen vissza, ha hibát talált.